index.html

```diff
@@ -84,22 +149,52 @@
     }
     </script>
     <script type="module">
-    import '@rhds/elements/rh-tooltip/rh-tooltip.js';
+      import '@patternfly/elements/pf-icon/pf-icon.js';
+      import '@rhds/elements/rh-tooltip/rh-tooltip.js';
+      import '@rhds/elements/rh-button/rh-button.js';
     </script>
+
+    <script type="module">
+      async function copy(event) {
+        const { html } = event.target.dataset;
+        await navigator.clipboard.writeText(html)
+      }
+
+      const tooltips = document.querySelectorAll('rh-tooltip');
+      tooltips.forEach(tip => {
+        const tipButton = tip.querySelector('rh-button');
+        tipButton.addEventListener('click', (event) => {
+          copy(event); 
+          tip.show();
+        })
+      })
+    </script>    
   </head>
   <body>
     <header><h1>Red Hat Icons</h1></header>
     <main>{% for set in iconSets %}
-      <section class="{{ set.set }}">
-        <h2>{{ set.set }}</h2>
+      {% set id = set.set | lower %}
```

Comment:
Add accessible toast for "Copied" after clicking the copy button. Probably display this in the upper right corner of the screen.

Don't forget to add `role="status"`, eg:

```html
<div class="toast" role="status">
  <p>Copied!</p>
</div>
```